modules/user_accounts/php/useraccount.class.inc

```diff
@@ -0,0 +1,63 @@
+<?php declare(strict_types=1);
+/**
+ * This class implements a data Instance which represents a single
+ * user in the user_accounts menu table.
+ *
+ * PHP Version 7
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\user_accounts;
+
+/**
+ * A User represents a row in the user_accounts menu table.
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class UserAccount implements \LORIS\Data\DataInstance
```

Comment:
Maybe this class should be called UserAccountRow instead?

Reply:
@driusan @ridz1208 what are your thoughts? Are we moving away from using `Row` as qualifiers for object instances? Or should keep `Row`?